### PR TITLE
Create Visualisations

### PR DESCRIPTION
**Pull request by Murphy-98**

### CODE REVIEW

File: Data/AKC Dog Breeds/main.csv [hunk 1]
```diff
@@ -0,0 +1,151 @@
+Breed,height_low_inches,height_high_inches,weight_low_lbs,weight_high_lbs
```
Comment (drvinceknight) [inline Data/AKC Dog Breeds/main.csv:1]: Let's remove the data files from version control but include information about how to get them in the respective READMEs.

File: src/R/main.R [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+##### visualising colourmaps #####
+
+ggplot(data.frame(x = rnorm(10000), y = rnorm(10000)), aes(x = x, y = y)) +
+geom_hex() + coord_fixed() +
+scale_fill_viridis(option='inferno') + theme_bw()
+
+##### Ninja Warrior data #####
+#Data from: https://data.world/ninja/anw-obstacle-history
+
+NinjaWarrior <- read.csv("C:/Users/Katie/OneDrive/Uni_Work_Year4/Project/Data/American Ninja Warrior Obstacle History.csv")
```
Comment (drvinceknight) [inline src/R/main.R]: Use a relative path.

File: src/R/main.R [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+obstacles <- function(ObstacleNumbers){
```
Comment (drvinceknight) [inline src/R/main.R]: Let's document these functions. See https://style.tidyverse.org/documentation.html

File: src/py/main.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#python -m pip install -U pip
+#python -m pip install -U matplotlib
+
+import matplotlib.pyplot as plt 
+import numpy as np
+import pandas as pd
+
+Dogs = pd.read_csv('C://Users//Katie//OneDrive//Uni_Work_Year4//Project//Data//AKC Breed Info.csv')
```
Comment (drvinceknight) [inline src/py/main.py]: Relative path.

File: src/py/main.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+#python -m pip install -U pip
+#python -m pip install -U matplotlib
+
```
Comment (drvinceknight) [inline src/py/main.py]: Not here.

File: src/R/main.R [hunk 6]
```diff
@@ -0,0 +1,123 @@
+##### Load libraries #####
+library(ggplot2)
+library(dplyr)
+library(DAAG) #datasets
+library(viridis) #colourmaps
```
Comment (drvinceknight) [inline src/R/main.R]:
```suggestion
library(ggplot2)
library(dplyr)
library(DAAG)
library(viridis)
```

File: src/R/main.R [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+# Barplot in default colour map 
+
+ggplot(data=obst) +
+geom_col(aes(x=Obstacle.Name, y=Times.Used, fill=Obstacle.Name)) +
+scale_x_discrete(labels=NULL) +
+xlab('Obstacle') + 
+ylab('Times Used') +
+labs(fill = 'Obstacle') +
+theme_classic()
```
Comment (drvinceknight) [inline src/R/main.R]: Let's save these plots to a directory.

Comment (drvinceknight) [reply src/R/main.R]: (It's often sensible not to include pdfs in version control but we will here.)

**Review by drvinceknight — CHANGES_REQUESTED** [review]
Some minor stylistic tweeks, and let us add a stylcheck to the CI. Once those are done I'll look through the code.

File: src/R/main.R (hunk 6 above)
Comment (drvinceknight) [reply src/R/main.R]: Don't use comments unless you need to: document instead. (We can discuss what that means.)

**Review by Murphy-98 — COMMENTED** [review]
NOTE: Found a sales dataset in R library 'datasets' package to take values from

**Pull request merged**